code/__DEFINES/sound_defines.dm

```diff
@@ -9,13 +9,14 @@
 #define CHANNEL_ENGINE 1017 // Engine ambient sounds
 #define CHANNEL_FIREALARM 1016 //fire alarm alarms
 #define CHANNEL_ASH_STORM 1015
+#define CHANNEL_DELTA_ALARM 1014
```

Comment:
again not a huge fan of these hyper specific channels. this isnt maintainable. we cant keep adding channels like this. might i suggest merging this into the firealarm sound and renaming that to "alarms"

Reply:
that was a holdover from when I was planning to make it muteable, i dropped that but left the half-baked channel thing. However, would putting it on alarm channel mean a firealarm could stop the alert alarm?

I don't have much experience with sound code

Reply:
Upon testing, putting the delta alarm on the fire alarm channel causes issues. Mainly that fire alarms interrupt the delta alarm and cause jank. Since the delta alarm needs to be independently stoppable (when the station explodes, when the alert level changes) it needs to be on a dedicated channel.